test-cases/README.md

```diff
@@ -187,33 +187,33 @@ It is also possible to set the Jira username and password in environment variabl
 Use the `export csv` cmd to list and export the test cases in csv:
 
 ```
-./tools.sh export csv
+./tools.sh export csv --product PRODUCT_NAME
```

Comment:
Not sure if we should always filter by-product, the product should be a column in the CSV not a mandatory filter in my opinion